Makefile

```diff
@@ -20,4 +20,14 @@ lint:
 
 check: lint test
 
-.PHONY: run build test clean install all check lint
+dev-init:
+	asdf install
+	npm install -g npm@latest
+	npm install -g yarn@latest
+	yarn install
+	echo 4202 > ~/.puma-dev/metrics.prx
```

Comment:
I feel like we've moved away from Makefiles at this point.  And these minimal commands could be included in the docs.  But then again, you're keeping things in line with the current codebase.